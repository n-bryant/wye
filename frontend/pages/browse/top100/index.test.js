import { TopHundredPage } from "./index";

describe("TopHundredPage", () => {
  const props = {
    calsses: {
      root: "root"
    }
  };

  it("should render successfully", () => {
    const wrapper = shallow(<TopHundredPage {...props} />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
