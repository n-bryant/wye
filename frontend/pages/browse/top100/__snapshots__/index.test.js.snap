// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopHundredPage should render successfully 1`] = `
<div
  className="wye-TopHundredPage"
>
  <WithWidth(WithStyles(GamesFilterWidget))
    initialFilters={
      Object {
        "filters": Object {
          "gameFilters": Object {
            "ownersMin_gte": 2000000,
            "userRating_gte": 75,
          },
        },
        "first": 100,
        "orderBy": Array [
          "PLAYTIME_RECENT",
          "USER_RATING",
        ],
        "sortOrder": "DESC",
      }
    }
    subtitle="Browse the most popular by recent playtime"
    title="Top 100"
  />
</div>
`;
